backend/spec/helpers/admin/base_helper_spec.rb

```diff
@@ -14,4 +14,30 @@
       expect(datepicker_field_value(date)).to eq("2013/08/14")
     end
   end
+
+  describe "#admin_layout" do
+    subject { admin_layout(value) }
+
+    context "when an initial value is set" do
+      let(:value) { "full-width" }
+
+      it "sets and returns the value passed to it" do
+        expect(subject).to eq value
+      end
+
+      context "and you call it again without an argument" do
+        before { admin_layout(value) }
+        let(:value) { nil }
+
+        it "returns the initial value that was set" do
+          expect(subject).to eq value
+        end
+      end
+    end
```

Comment:
I think there's something wrong with the test, as I think on line 33 you are essentially saying 

```ruby
expect(admin_layout(nil)).to eq nil
```
because the `let` is run before the `before` which is the same as the last test, and that's why it passes.

It's just that the over-ambitious  `subject {}` gets in the way of making the tests much simpler. The tests are also a bit difficult to follow due to this `let(:value)` which keeps changing. I'd rather do one `describe "#admin_layout" do` followed by 3 tests with simple `it`, and in each it, do directly  
```ruby
admin_layout('full-width')
expect(admin_layout). to eq('full-width')
```
- you will only be gaining simplicity